lms/static/karma_lms.conf.js

```diff
@@ -28,7 +28,6 @@ var options = {
     sourceFiles: [
         {pattern: 'coffee/src/**/!(*spec).js'},
         {pattern: 'course_bookmarks/**/!(*spec).js'},
-        {pattern: 'course_experience/js/**/!(*spec).js'},
```

Comment:
No longer need to explicitly load the source, because the test file imports it so it will be included in the bundle by Webpack.